airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -319,6 +320,27 @@ def post_dag_run(*, dag_id: str, session: Session = NEW_SESSION) -> APIResponse:
                 dag_hash=get_airflow_app().dag_bag.dags_hash.get(dag_id),
                 session=session,
             )
+            from airflow.models.log import Log
```

Comment:
Instead of this, there's `action_logging` decorator you should use, see https://github.com/apache/airflow/blob/5d84b59554c93fd22e92b46a1061b40b899a8dec/airflow/api_connexion/endpoints/connection_endpoint.py#L51

Reply:
@ephraimbuddy thanks, my dev env was a bit outdated so I didn't have this decorator.
Updated and changed the way for db insertion. Please review.